test/remote/db_tests.js

```diff
@@ -661,6 +662,16 @@ describe('remote db', function() {
     }
   )
 
+  afterEach(() => {
+    return db.emailRecord(ACCOUNT.email)
+      .then(function (emailRecord) {
+        return db.deleteAccount(emailRecord)
+      })
+      .catch(function (err) {
+        //
```

Comment:
For completeness, it may be worth re-throwing this error if it's not something like UNKNOWN_ACCOUNT, just in case.